### DISCUSSION TITLE
Have you ever tried to prove the vendors claims for transferring data?

### DISCUSSION THREAD

**Discussion by drt1** [Hypfer/Valetudo#2068] [Q&A / Support]
### Are you already using Valetudo?

No, but generally interested with no specific timeframe

### Your questions

Hey folks,
as the vendors claim that camera data will be processed directly on the robot and not sent through the internet and so on:
Have you ever tried to prove that (wrong)?

**Comment by Hypfer** [comment]
I recommend watching Dennis' talks:
https://dontvacuum.me/talks/
